tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -48,6 +48,7 @@ struct testing_callbacks {
     int expected_precision               = -1;
     size_t expected_dynamic_precision    = static_cast<size_t>(-1);
     bool expected_auto_dynamic_precision = false;
+    bool expected_localized              = false;

```

Comment:
Nothing sets this - I believe we should have basic parsing coverage in
https://github.com/microsoft/STL/blob/ee5a216b516ef033d9153b324842369eee7653d1/tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp#L140-L142
In particular, we should cover both:
* `L` is followed by *chrono-specs*
* `L` is at the end, not followed by *chrono-specs*